Linter: copy multi level ApiModules to the lint dir (#2942)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **doc-review** required dot suffixes in release notes describing new content.
* Improved logs and error handling in the **modeling-rules test** command.
* Fixed an issue where **download** command failed when running on a beta integration.

> * Fixed an issue where **lint** would not properly handle imports of multi level ApiModules.

Reviewer: ```suggestion
* Added support for code importing multi-level ApiModules to **lint**.
```

it's a feature, not a bug :)
